subsys/mgmt/mcumgr/lib/README.md

```diff
@@ -0,0 +1,137 @@
+# mcumgr
+
+This is mcumgr, version 0.2.0
+
+mcumgr is a management library for 32-bit MCUs.   The goal of mcumgr is to
+define a common management infrastructure with pluggable transport and encoding
+components.  In addition, mcumgr provides definitions and handlers for some
+core commands: image management, file system management, and OS management.
+
+mcumgr is operating system and hardware independent.  It relies on hardware
+porting layers from the operating system it runs on.  Currently, mcumgr runs on
+both the Apache Mynewt and Zephyr operating systems.
+
+## Getting started
+
+For tips on using mcumgr with your particular OS, see the appropriate file from
+the list below:
```

Comment:
probably should be re-worded because Zephyr is our only OS.

Reply:
Removed